### PR TITLE
fix: presets-demo and select-field styles

### PR DESCRIPTION
**Pull request by appsparkler**
Fixes:
- presets demo wasn't working
- minor style issue when you select the `bootstrap` in the field for selecting presets.

### CODE REVIEW

File: packages/docs/src/components/presets-demo.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default function PresetsDemo() {
           <span>Preset:</span>
           <Select
             id="theme"
-            sx={{ display: 'inline-flex' }}
+            sx={{ display: 'inline-flex', paddingRight: 24 }}
```
Comment (lachlanjc) [inline packages/docs/src/components/presets-demo.js]: Good catch here. Let's increase it further though? On my iPhone:

image

File: packages/docs/src/components/presets-demo.js [hunk 2]
```diff
@@ -16,7 +15,8 @@ import Lorem from './lorem.mdx'
 
 export default function PresetsDemo() {
   const [theme, setTheme] = useState('base')
-  const preset = presets[theme]
+  const preset = presets[theme]   
+
```
Comment (lachlanjc) [inline packages/docs/src/components/presets-demo.js:19]:
```suggestion
```

**Review by lachlanjc — APPROVED** [review]
Thank you for the contribution! Looks great.

File: packages/docs/src/components/presets-demo.js (hunk 1 above)
Comment (hasparus) [inline packages/docs/src/components/presets-demo.js]:
```suggestion
            sx={{ display: 'inline-flex', paddingRight: 32 }}
```

**Comment by hasparus** [comment]
Does anybody have an idea why Cypress fails here?

File: packages/docs/src/components/presets-demo.js (hunk 1 above)
Comment (appsparkler) [reply packages/docs/src/components/presets-demo.js]: @lachlanjc , I've added the style to the core `Select` component : [chore: update style for core Select component](https://github.com/system-ui/theme-ui/pull/2058)

**Pull request merged**